spec/models/sponsored_benefits/employer_profile_builder_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 module SponsoredBenefits
   RSpec.describe BenefitApplications::EmployerProfileBuilder, type: :model do
-    let(:effective_period_start_on) { TimeKeeper.date_of_record.end_of_month + 1.day + 1.month }
+    let(:effective_period_start_on) { TimeKeeper.date_of_record.end_of_month + 1.day }
```

Comment:
@jacobkagon Why can't we make effective_period_start_on as TimeKeeper.date_of_record.beginning_of_month? and in your 'it' block, you can expect different things if the system is under OE or non-OE.